torch/nn/modules/pooling.py

```diff
@@ -599,6 +602,12 @@ class AvgPool2d(_AvgPoolNd):
               W_{out} = \left\lfloor\frac{W_{in}  + 2 \times \text{padding}[1] -
                 \text{kernel\_size}[1]}{\text{stride}[1]} + 1\right\rfloor
 
+          If ``ceil_mode`` is True and :math:`(H_{out} - 1)\times \text{stride}[0]\geq H_{in}
+          + \text{padding}[0]`, we skip the last window, resulting in :math:`H_{out}` being reduced by one.
+
+          Same for :math:`W_{out}`, if ``ceil_mode`` is True and :math:`(W_{out} - 1)\times
+          \text{stride}[1]\geq W_{in} + \text{padding}[1]`, :math:`W_{out}` is reduced by one.
```

Comment:
nit: For less verbosity I think we can just say "The same applies for :math:`W_{out}`"

Reply:
Done, thanks!